### PR TITLE
AA-595: Add script to delete Braze user information

### PR DESCRIPTION
**Pull request by mikix**
Adds two new config variables:
- braze_api_key: a REST API key from Braze with users.delete access
- braze_instance: a Braze instance key like iad-01

And sets the BRAZE service variable & 'delete_user' method, for use in retirement pipelines.

### CODE REVIEW

File: tubular/scripts/helpers.py [hunk 1]
```diff
@@ -183,6 +187,12 @@ def _setup_all_apis_or_exit(fail_func, fail_code, config):
 
         config['LMS'] = LmsApi(lms_base_url, lms_base_url, client_id, client_secret)
 
+        if braze_api_key:
+            config['BRAZE'] = BrazeApi(
+                braze_api_key,
+                braze_instance,
+            )
```
Comment (Dillon-Dumesnil) [inline tubular/scripts/helpers.py:194]: Nit: one line?

File: tubular/braze_api.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+class BrazeApi:
+    """
+    Braze API client used to make calls to Braze
+    """
+
+    def __init__(self, braze_api_key, braze_instance):
+        self.api_key = braze_api_key
+
+        # https://www.braze.com/docs/api/basics/#endpoints
```
Comment (Dillon-Dumesnil) [inline tubular/braze_api.py:30]: 👏

**Review by Dillon-Dumesnil — APPROVED** [review]
LGTM

File: tubular/scripts/helpers.py (hunk 1 above)
Comment (mikix) [reply tubular/scripts/helpers.py:194]: Fair nit, but all the other stanzas just like this were doing one-arg-per-line, so I figured consistency was better here.

**Pull request merged**